stories/Organism/Sidebar/sidebar.scss

```diff
@@ -0,0 +1,166 @@
+@import '../../assets/scss/mixins';
+@import '../../assets/scss/breakpoints';
+@import '../../assets/scss/variables';
+
+.accordion-wrapper {
+  .accordion-navbar {
+    display: none;
+
+    @include devicebreak(medium) {
+      display: block;
+      padding: 0;
+    }
+  }
+
+  &.show {
+    .accordion-navbar {
+      display: block;
+      padding: 0;
+    }
+
+    .accordion-menu::before {
+      @include transform(rotateZ(180deg));
+    }
+  }
+}
+
+.accordion-menu {
+  display: block;
+  font-family: $Font-Family-ProximaBold;
+  font-size: $font-size-16;
+  padding: $spacing-16 0 $spacing-24 $spacing-24;
+  text-transform: uppercase;
+
+  @include devicebreak(medium) {
+    display: none;
+  }
+}
+
+.accordion-wrapper {
```

Comment:
@pallavikadam-srijan you have also define accordion-wrapper on the top also so can you please put all styling in one place only and please follow the standard like accordion and then using child like &__menu